src/metal.cs

```diff
@@ -393,10 +402,16 @@ partial interface MTLCommandBuffer {
 		void PopDebugGroup ();
 
 #if NET
+#if !TVOS
+		[MacCatalyst (14, 0), Mac (11, 0), iOS (13, 0), NoTV]
 		[Abstract] // @required but we can't add abstract members in C# and keep binary compatibility
+#else
+		[NoMacCatalyst, NoMac, NoiOS, TV (16,0)]
+#endif
```

Comment:
I believe this should be the pattern:

```suggestion
#if XAMCORE_5_0
		[MacCatalyst (14, 0), Mac (11, 0), iOS (13, 0), TV (16, 0]
		[Abstract] // @required but we can't add abstract members in C# and keep binary compatibility
#elif !TVOS
		[MacCatalyst (14, 0), Mac (11, 0), iOS (13, 0), NoTV]
		[Abstract] // @required but we can't add abstract members in C# and keep binary compatibility
#else
		[NoMacCatalyst, NoMac, NoiOS, TV (16,0)]
#endif
```